src/plugins/Utilities/SConscript

```diff
@@ -14,5 +14,5 @@ SConscript(dirs=subdirs, exports='env osname', duplicate=0)
 # Optional targets
 optdirs = ['danahddm', 'dumpcandidates', 'dumpthrowns', 'l3bdt']
 optdirs.extend(['merge_rawevents', 'syncskim', 'DAQ', 'TTab', 'rawevent'])
-optdirs.extend(['cdc_emu', 'fmwpc_scan', 'epem_ml_skim'])
+optdirs.extend(['cdc_amp_t','cdc_echo','cdc_emu', 'fmwpc_scan', 'epem_ml_skim'])
```

Comment:
cdc_echo cannot be a default target and an optional target at the same time

Reply:
Sorry.

Reply:
I removed cdc_echo from the default targets, as it serves only a 'specific use case'